src/convertToObject.js

```diff
@@ -5,8 +5,27 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter((declaration) => declaration !== '')
+    .map((declaration) => {
+      const propertyValuePair = declaration.split(':');
+
+      propertyValuePair[0] = propertyValuePair[0].trim();
+      propertyValuePair[1] = propertyValuePair[1].trim();
+
+      return propertyValuePair;
+    })
+    .reduce((accumulator, propertyValuePair) => {
```

Comment:
The `reduce` method is correctly used to transform an array of property-value pairs into an object. Make sure to handle potential edge cases, such as when there are multiple colons in the declaration. Currently, your code will not correctly handle such cases.

Reply:
ok